### DISCUSSION TITLE
[Unintended Feature] Highlight arround the summary superfence

### DISCUSSION THREAD

**Discussion by SPyofgame** [facelessuser/pymdown-extensions#1995] [Q&A] [answered]
I just notice that this sometimes happens (especially spamming F5), but I dont really know how is it produced. I will change this discussion to be more formal and details if necessary.

![image](https://user-images.githubusercontent.com/62207095/226676080-19fbf33b-4a18-4ebe-975a-5a1eddc75b34.png)

**Comment by facelessuser** [comment, marked as answer]
Highlighting is a theme issue and has nothing to do with what our extensions generate as we do not provide styling in our output. IIRC, it is likely for accessibility in the Material theme, but if it is thought to be a problem, I would ask over there.

**Reply by SPyofgame** [reply]
Seems like pressing `space` after opening a detail made it happen, is it intended?

**Reply by facelessuser** [reply]
Browsers control how users interact with the elements, not us, and theming controls how the elements look. JavaScript can also intercept actions and perform additional things.

MkDocs Material has some accessibility theming. When you tab to detail, it gets a border because it is focused. If you click a details and then hit space, it causes the details to get focused, and then the accessibility theming kicks in. If you don't like it, you'll have to add your own custom CSS.

**Reply by SPyofgame** [reply]
oh ok 🤔
